spec/05_select_random_elements_from_array_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Specification for RubyMonk Primer Problem 5 Select random elements from an array
+# Note that the tests implemented here are not the same as Ruby Monk's
+
+require_relative '../05_select_random_elements_from_array'
+
+RSpec.describe SelectRandomElementsFromArray do
+  subject(:array_elements) { described_class.new }
+
+  describe '#random_select' do
+    it 'passing [1, 2, 3, 4, 5] and 2 returned' do
+      expect(array_elements.random_select([1, 2, 3, 4, 5], 2).count).to be 2
+    end
+
+    it 'returns only the values present in the array passed to the method' do
+      og_array = [1, 2, 3, 4, 5]
+      results = array_elements.random_select(og_array, 3)
+      expect(Set.new(results).subset?(Set.new(og_array))).to be true
+    end
+
+    it 'does not call Array#sample' do
+      og_array = [99, 23, 33, 104, 57, 4444]
+      # rubocop:disable RSpec/MessageSpies
```

Comment:
You might be able to appease The Cop by adding `before { allow_any_instance_of(Array).to receive(:sample) }`, but that's a whole lotta weird given that you care about the opposite.  I also dislike `any_instance_of` when it can be avoided.  So TL;DR ignoring 👮‍♂️ here is a good decision in my book.

Reply:
Thank you for this because when I tried appeasing Rubocop it just felt like it was not was I was trying to test.